### ISSUE TITLE
Elevated urine mevalonate levels

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5168]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC442441

**Comment by pnrobinson** [comment]
@Marwaabdelhakim

**Comment by pnrobinson** [comment]
@Marwaabdelhakim

**Comment by pnrobinson** [comment]
mevalonate
CHEBI:25350
 Definition | A hydroxy monocarboxylic acid anion  that is the conjugate base of mevalonic acid, arising from deprotonation  of the carboxy group.

So the proper parent is https://hpo.jax.org/app/browse/term/HP:0040156

**Comment by pnrobinson** [comment]
Note -- added as HP:0032638

For consistency, I used  Elevated urinary mevalonic acid as the primary label land added mevanolate (etc) as synonyms.
Characterizes OMIM:610377
Def: An abnormally increased amount of mevanolate in the urine. Mevanolate is that hydroxy monocarboxylic acid anion that is the conjugate base of mevalonic acid.
